packages/utils/test/testutils.ts

```diff
@@ -0,0 +1,13 @@
+export const testOnlyIfNodeVersionAtLeast = (minVersion: number): jest.It => {
+  const currentNodeVersion = process.env.NODE_VERSION;
+
+  try {
+    if (Number(currentNodeVersion?.split('.')[0]) < minVersion) {
```

Comment:
Can we use `parseSemver` from `@sentry/utils` here?

Reply:
We could (and thanks for pointing out that that exists - TIL!), but it feels like overkill, and likely to (at least slightly) slow things down. We know for sure that the node version will be `<number>.<otherstuff>` (so I think it's safe to rely on built-ins) and really that first number is all we need (so the overhead of calling a function, creating and using a regex, etc, etc doesn't seem worth it).